ruby.c

```diff
@@ -272,6 +272,7 @@ usage(const char *name, int help)
 	M("s",     ", save-temps",     "Save MJIT temporary files in $TMP or /tmp"),
 	M("w",     ", warnings",       "Enable printing MJIT warnings"),
 	M("d",     ", debug",          "Enable MJIT debugging (very slow)"),
+	M("a=num", ", aot=num",        "Ahead of Time Compileation after num calls"),
```

Comment:
typo: Compileation

Reply:
Thank you https://github.com/ruby/ruby/pull/1782/commits/a87c8ef5319a020aeeca241f4ac05dbc60c89367